Library/Formula/lcm.rb

```diff
@@ -0,0 +1,26 @@
+class Lcm < Formula
+  desc "libraries and tools for message passing and data marshalling"
+  homepage "https://lcm-proj.github.io/"
+  url "https://github.com/lcm-proj/lcm/releases/download/v1.2.1/lcm-1.2.1.zip"
+  sha256 "55b959409fc1c0cfa705209b679dc4b3043b68f14078e192b65c1b6021c37ea6"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on :java => :recommended
+  depends_on :python => :recommended
```

Comment:
Should be:

```ruby
depends_on :python => :recommended if MacOS.version <= :snow_leopard
```
That way we don't enforce the use of Homebrew's Python on everyone who wants to install this formula from our binary packages.